package.json

```diff
@@ -84,5 +84,10 @@
     "Matt Dobson <mdobson@apigee.com> (https://github.com/mdobson)",
     "Mitch Fierro <fierro.mitch@gmail.com> (https://github.com/f1erro)"
   ],
-  "author": "Kevin Swiber <kswiber@gmail.com>"
+  "author": "Kevin Swiber <kswiber@gmail.com>",
+  "engines" : { 
+     "node" : ">= 4.8.2 <=6.9.1",
```

Comment:
Should probably to look to update this to something like `>=6.9.1 <=7.9.0` This is the latest node.